tests/macos_cpp_bugs.txt

```diff
@@ -13,3 +13,5 @@ cpp_stl_algo_permutation_ops
 cpp_stl_algo_sorted_ranges_set_ops
 cpp_stl_algo_sorted_ranges_other_ops
 cpp_stl_bit_cpp20
+cpp_stl_cmath_cpp20
+cpp_stl_cmath_cpp17
```

Comment:
If `std::beta()` is the issue here, then could you use a different function in this test? Then we could at least enable the C++17 tests on macOS.

Reply:
I double-checked it on my Mac and it turned out Apple Clang doesn't support the C++17 mathematical special functions yet, see [here](https://en.cppreference.com/w/cpp/compiler_support/17). :/ The only exception is the trivial `hypot` function.

Reply:
Not just apple clang, but clang itself. It's not around the corner [either](https://reviews.llvm.org/D142806).